lib/Person.rb

```diff
@@ -0,0 +1,38 @@
+require 'pry'
+
+class Person
+  def self.get_user_name
+    puts "What's your name?"
+    gets.chomp.capitalize
+  end
+
+  def self.get_user_age
+    puts"How old are you?"
+    gets.chomp.to_i
+  end
+
+  def self.get_user_gender
```

Comment:
A sign this belongs somewhere else: you use a different class name in the variable name
`get_user_name` --> put this in `User` and call it `get_name`